src/ucp/rndv/proto_rndv.inl

```diff
@@ -113,6 +113,7 @@ static UCS_F_ALWAYS_INLINE size_t ucp_proto_rndv_rts_pack(
         rkey_size    = 0;
     } else {
         rts->address = (uintptr_t)req->send.state.dt_iter.type.contig.buffer;
+        req->flags  |= UCP_REQUEST_FLAG_RKEY_INUSE;
```

Comment:
can we set it inside `ucp_proto_request_pack_rkey`?

Reply:
Done.